### PR TITLE
Removed total views from the usercard

### PR DESCRIPTION
**Pull request by mmattbtw**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description
Twitch removed the 'total views' counter from the channel page, dashboard, and I am certain the number isn't being updated anymore. [Announcement of total views being deprecated](https://discuss.dev.twitch.tv/t/get-users-api-endpoint-view-count-deprecation/37777)

### CODE REVIEW

**Comment by neil-lobo** [comment]
https://discuss.dev.twitch.tv/t/get-users-api-endpoint-view-count-deprecation/37777

**Comment by iProdigy** [comment]
Also consider removing the field from the underlying data models

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 - Minor: Added `/requests` command. Usage: `/requests [channel]`. Opens the channel points requests queue for the provided channel or the current channel if no input is provided. (#3746)
 - Minor: Added ability to execute commands on chat messages using the message context menu. (#3738, #3765)
 - Minor: Added `/copy` command. Usage: `/copy <text>`. Copies provided text to clipboard - can be useful with custom commands. (#3763)
+- Bugfix: Removed total views from the usercard. (#3792)
```
Comment (pajlada) [inline CHANGELOG.md:23]: Not a bugfix, should be classified as a Minor change

**Review by pajlada — CHANGES_REQUESTED** [review]
Please remove the now unused `ui_.viewCountLabel` member in `UserInfoPopup`
Also remove the `viewCount` member from the model

**Review by pajlada — APPROVED** [review]

**Review by iProdigy — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@
 - Minor: Added `/requests` command. Usage: `/requests [channel]`. Opens the channel points requests queue for the provided channel or the current channel if no input is provided. (#3746)
 - Minor: Added ability to execute commands on chat messages using the message context menu. (#3738, #3765)
 - Minor: Added `/copy` command. Usage: `/copy <text>`. Copies provided text to clipboard - can be useful with custom commands. (#3763)
+- Minor: Removed total views from the usercard. (#3792)
```
Comment (Felanbird) [inline CHANGELOG.md:23]:
```suggestion
- Minor: Removed total views from the usercard, as Twitch no longer updates the number. (#3792)
```
sneaking in a quick blame towards Twitch so we don't get blamed 🙂

**Review by Felanbird — APPROVED** [review]

**Pull request merged**